WalletWasabi.Packager/Program.cs

```diff
@@ -272,6 +274,7 @@ private static async Task PublishAsync()
 				$"--runtime \"{target}\"",
 				$"--disable-parallel",
 				$"--no-cache",
+				$"--no-restore",
```

Comment:
Inspired by https://github.com/NuGet/Home/issues/8287#issuecomment-507454556 otherwise we can't publish at all.

Reply:
Hmm, where was the restoration done? How RestoreLockedMode=true is working in this case?

Reply:
> Hmm, where was the restoration done? 

I believe it basically says "use packages.lock.json" files as is without attempting to modify it for that one RID we want to publish (that would modify `packages.lock.json` files). `dotnet restore` should be only necessary when you change something but here we want to publish using what we have. 

> How RestoreLockedMode=true is working in this case?

To build, one needs to know the dependencies, so all the information is taken from `packages.lock.json` files and given that implicit restore is disabled there is no attempt to recompute it again. So we basically say that "we rely on `packages.lock.json` files to be correct which seems like a reasonable assumption.

This is how I understand it.

Reply:
I fixed it - added restore back separately.

Reply:
haha, yeah, it's nuanced as hell :)